fdbclient/FDBOptions.h

```diff
@@ -42,15 +42,25 @@ struct FDBOptionInfo {
 	// be no cumulative effects from calling multiple times).
 	int defaultFor;
 
+	enum FDBOptionParamType{
```

Comment:
I'd recommend using `enum class` here, which will likely also require a cast when converting the integer to the enum type. Also, we can optionally drop the `FDBOption` from the name given that it's within the `FDBOptionInfo` scope now.

```suggestion
	enum class ParamType{
```

Reply:
@sfc-gh-chliu This is not a required change and I'll leave it up to you whether you want it. It might make things a little less verbose to have `FDBOptionInfo::ParamType` rather than `FDBOptionInfo::FDBOptionParamType`.